controller/settingcontroller.php

```diff
@@ -13,6 +13,7 @@
 namespace OCA\Music\Controller;
 
 use \OCP\AppFramework\Controller;
+use OCP\AppFramework\Http;
```

Comment:
isn't here a leading '\' missing so that the line instead should be written like
```php
use \OCP\AppFramework\Http;
```
?